docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -12,9 +12,12 @@ import { getComponentPathname, typeOrder, repoURL } from 'docs/src/utils'
 
 const pkg = require('../../../../package.json')
 const componentMenu = require('docs/src/componentMenu')
+const behaviorMenuItems = require('docs/src/componentMenuBehaviors')
 
 const selectedItemLabelStyle: any = { color: '#35bdb2', float: 'right' }
 const selectedItemLabel = <span style={selectedItemLabelStyle}>Press Enter</span>
+// assume all behaviors are the same type
+const behaviorType = behaviorMenuItems[0].type
```

Comment:
this is a dangerous assumption, the types might be different, or if all of them are the same, no need to introduce the type at all and this construction can also cause `undefined` error